advanced/is-pattern-1/README.md

```diff
@@ -83,7 +83,7 @@ please provide your WSO2 Subscription Credentials via input values (using `--set
 Refer the following example.
 
 ```
- helm install --name <RELEASE_NAME> wso2/is-pattern-1 --version 5.11.0-3 --namespace <NAMESPACE> --set wso2.subscription.username=<SUBSCRIPTION_USERNAME> --set wso2.subscription.password=<SUBSCRIPTION_PASSWORD>
+ helm install <RELEASE_NAME> wso2/is-pattern-1 --version 5.11.0-3 --namespace <NAMESPACE> --set wso2.subscription.username=<SUBSCRIPTION_USERNAME> --set wso2.subscription.password=<SUBSCRIPTION_PASSWORD> --set wso2.deployment.wso2is.imageTag=<SUBSCRIPTION_VERSION>
 ```
```

Comment:
Added sections for Helm 2 & 3

@chirangaalwis the version is necessary since the version naming strategy is different from public to private containers (5.11.0 = 5.11.0.0). This may not be intentional but what I have found. Industry standard would be to just drop the 0s in both repos for the stable version. (5.11, 5.11-alpine, 5.11-centos)

Reply:
The versioning strategy in the public DockerHub and the private Docker registry repositories differ primarily based on the presence of WSO2 Updates.

DockerHub images need not indicate the update levels as these images package the plain GA product pack versions. But with the private registry, this differs as it is required to indicate the WSO2 Update levels.

@Aaquiff have we handled the usage of the latest WSO2 Update level tag in our WSO2 product Helm resources? If so, we don't need to explicitly define the above image tag.

Reply:
the `imageTag: "5.11.0"` is what is in the values.yaml. `Error response from daemon: manifest for docker.wso2.com/wso2is:5.11.0 not found: manifest unknown: manifest unknown`

it will not work unless overwritten.

Reply:
@vordimous let's change it to `5.11.0.0`.